File: 461.HammingDistance/Solution.py
class Solution:
    def hammingDistance(self, x, y):
        """
        :type x: int
        :type y: int
        :rtype: int
        """
        count = 0
        for i in range(31):
            bin_x, bin_y = x%2, y%2
            if bin_x != bin_y:
                count += 1
            x, y = x//2, y//2
        return count
                
        
        

File: 461.HammingDistance/Solution.java
class Solution {
    public int hammingDistance(int x, int y) {
        int count = 0;
        for(int i=0; i<31; i++){
            int bin_x = x%2;
            int bin_y = y%2;
            if(bin_x !=bin_y){
                count++;
            }
            x>>=1;
            y>>=1;
        }
        return count;
    }
}